Ardupilog.m

```diff
@@ -278,13 +277,6 @@ function delete(obj)
                 newFmt = char(trimTail(msgData(7:22)));
                 newLabels = char(trimTail(msgData(23:86)));
                 
-                % Create dynamic property of Ardupilog with newName, and add to logMsgGroups array
-                if isempty(obj.logMsgGroups)
-                    obj.logMsgGroups = addprop(obj, newName);
-                else
-                    obj.logMsgGroups(end+1) = addprop(obj, newName);
-                end
-                
                 % Instantiate LogMsgGroup class named newName, process FMT data
                 obj.(newName) = LogMsgGroup(newType, newName, newLen, newFmt, newLabels);
```

Comment:
The commit should appear on origin. Can you see it in your local repo?
If you committed while being in detached HEAD by accident (you should be on slicing-patch), then your commit won't be visible.
It may be there somewhere but I can't tell you how to bring it to life.